src/LicenseHeader.cs

```diff
@@ -0,0 +1,30 @@
+using AutoRest.TypeScript.DSL;
+
+namespace AutoRest.TypeScript
+{
+    class LicenseHeader
+    {
+        public static string GenerateLicenseHeaderTS()
+        {
+            if (AutoRest.Core.Settings.Instance.Header != "")
+            {
+                TSBuilder builder = new TSBuilder();
+                builder.Comment(AutoRest.Core.Settings.Instance.Header);
+                builder.Line();
+                return builder.ToString();
+            }
+            return "";
+        }
+        public static string GenerateLicenseHeaderJS()
```

Comment:
Does it make any difference if the comment is generated using TSBuilder vs JSBuilder? AFAIK `TSBuilder` would use `JSBuilder`'s (its parent) implementation of both `Line` and `Comment`.

I think you should be able to just keep one `GenerateLicenseHeader` method and use either `JSBuilder` or `TSBuilder`. Since at the end the method returns a `string` which one you chose shouldn't make a difference

Reply:
That makes sense--I noticed that there was a difference at the call-site, and didn't know if they behave differently under the hood. I can re-combine them

Reply:
There is one difference at the callsites--the JS portion did not include the `builder.Line` after `builder.Comment`. I still went ahead and combined them, and added back in the call to `builder.Line` after all the callsites that had it before.